packages/brick_offline_first/lib/src/offline_queue/graphql/graphql_offline_queue_link.dart

```diff
@@ -22,35 +23,43 @@ class GraphqlOfflineQueueLink extends Link {
     final cacheItem = GraphqlRequestSqliteCache(request);
     _logger.finest('sending: ${cacheItem.toSqlite()}');
 
-    final db = await requestManager.getDb();
-    // Log immediately before we make the request
-    await cacheItem.insertOrUpdate(db, logger: _logger);
+    // Ignore "mutation" and "subscription" request
+    if (!isMutation(cacheItem.request)) {
```

Comment:
Hang on - we want to be tracking only mutations.
```suggestion
    // Ignore "mutation" and "subscription" request
    if (isMutation(cacheItem.request)) {
```